src/commonComponents/dishImage/DishImage.js

```diff
@@ -0,0 +1,24 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Icons
+import vegIcon from './images/veg.png';
+import nonVegIcon from './images/nonveg.png';
+function DishImage(props) {
```

Comment:
this component doesn't look write, you should have a component img for this which takes icon as a prop which will change conditionally